### ISSUE TITLE
Portal2 crash when changing the volume with laptop media keys

### ISSUE THREAD

**Issue by marcosps** [ValveSoftware/portal2#238]
Hi guys, as I described in the subject, it crashes everytime I change the volume with media keys. I'm using Fedora 20 and Gnome3. 

System information sayd that I have:

Sound card:
    Audio device: Intel Haswell HDMI

Please tell me if you need additional info.

**Comment by ngoquang2708** [comment]
Oh God. I face this bug too. My system is standard Ubuntu 15.10 installation. When changing the volume in game using Fn + Up/Down, the bubble popup show the volume meter then the sound come to loop, and I can not do anything else except Ctrl + Alt + F1, login and restart lightdm.

Not only Portal 2, it's seem that any Source games are affected.

**Comment by codingcatgirl** [comment]
This happens for me too. 

SystemArch Linux.
Processor Information:
    Vendor:  AuthenticAMD
    CPU Family:  0x10
    CPU Model:  0x4
    CPU Stepping:  0x3
    CPU Type:  0x0
    Speed:  3200 Mhz
    4 logical processors
    4 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Unsupported
    SSE4a:  Supported
    SSE41:  Unsupported
    SSE42:  Unsupported

Network Information:
    Network Speed:  

Operating System Version:
    Linux (64 bit)
    Kernel Name:  Linux
    Kernel Version:  4.2.5-1-ARCH
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11800000
    X Window Manager:  GNOME Shell
    Steam Runtime Version:  steam-runtime-release_2015-06-12

Video Card:
    Driver:  NVIDIA Corporation GeForce GTX 550 Ti/PCIe/SSE2

```
Driver Version:  4.5.0 NVIDIA 358.16
OpenGL Version: 4.5
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 60 Hz
VendorID:  0x10de
DeviceID:  0x1244
Number of Monitors:  1
Number of Logical Video Cards:  1
Primary Display Resolution:  1920 x 1080
Desktop Resolution: 1920 x 1080
Primary Display Size: 18,78" x 10,55"  (21,54" diag)
                                        47,7cm x 26,8cm  (54,7cm diag)
Primary Bus: PCI Express 16x
Primary VRAM: 1024 MB
Supported MSAA Modes:  2x 4x 8x 16x 
```

Sound card:
    Audio device: 

Memory:
    RAM:  7986 Mb

Miscellaneous:
    UI Language:  English
    LANG:  de_DE.UTF-8
    Microphone:  Not set
    Total Hard Disk Space Available:  1819991 Mb
    Largest Free Hard Disk Block:  675408 Mb

Installed software:

Recent Failure Reports:
    Fri Jan  1 13:21:02 2016 GMT: file ''/tmp/dumps/assert_20160101141901_1.dmp'', upload yes: ''CrashID=bp-969c8f1c-6148-4b68-a9ed-d56022160101''
    Fri Jan  1 14:01:52 2016 GMT: file ''/tmp/dumps/crash_20160101145749_1.dmp'', upload yes: ''CrashID=bp-65d035bc-6b7e-4524-909c-b9e482160101''
    Fri Jan  1 14:04:58 2016 GMT: file ''/tmp/dumps/crash_20160101150221_1.dmp'', upload yes: ''Discarded=1''
    Fri Jan  1 14:08:29 2016 GMT: file ''/tmp/dumps/crash_20160101150511_1.dmp'', upload yes: ''Discarded=1''
    Fri Jan  1 14:19:06 2016 GMT: file ''/tmp/dumps/crash_20160101150840_1.dmp'', upload yes: ''Discarded=1''

**Comment by marcosps** [comment]
Closing this as it doesn't seem relevant anymore.
